proto/sifnode/clp/v1/types.proto

```diff
@@ -25,15 +25,39 @@ message Pool {
     (gogoproto.nullable) = false,
     (gogoproto.moretags) = "yaml:\"pool_units\""
   ];
-  string swap_price_native = 5 [
+  string external_liabilities = 5 [
```

Comment:
See docs for breaking proto changes, I don't think we should be changing field ids.

Reply:
@timlind I've tested using a localnet with existing chain data then updated the field IDs and nothing broke, the structure fields were still using their relevant values. Is it really a concern?

Reply:
my test followed those steps:
1. create a chain with current margin proto structure and field IDs (init and run chain) and create pools
2. stop chain
3. update proto/sifnode/clp/v1/types.proto Pool message struct to use different fields IDs
4. call make proto-all
5. call make install
6. run chain again
7. retrieve existing pools
8. pool field values are the same than what they were before the code changes

```
height: "188"
pool:
  external_asset:
    symbol: cusdc
  external_asset_balance: "488439008293"
  external_custody: "1000000000000000000"
  external_liabilities: "0"
  health: "0.999999989602279511"
  interest_rate: "0.600000000000000000"
  native_asset_balance: "96174925132652547206383901"
  native_custody: "5078652337075426"
  native_liabilities: "1000000000000000000"
  pool_units: "96174925423929435353999282"
  reward_period_native_distributed: "0"
  swap_price_external: "196.902629601115062319"
  swap_price_native: "0.005078652337075426"
```